src/vivarium_inputs/core.py

```
@@ -222,6 +216,10 @@ def get_exposure(
     data = extract.extract_data(entity, "exposure", location_id)
     data = data.drop("modelable_entity_id", "columns")
 
+    # Fixme: why are different level locations carrying this column along?
+    if "model_version_id" in data.columns:
+        data = data.drop("model_version_id", "columns")
+
```

Would it be reasonable to check if the column has only one value -- i.e., avoid a case where you have multiple `model_version_id`s?

---

I think we need to understand where/why it's getting dropped and consider fixing it at that point (basically you're 'fixme').

Regardless, slightly cleaner if you merge these new lines with the previous drop like

data = data.drop(["modelable_entity_id", "model_version_id"], "columns", errors="ignore")

---

We could check if there are multiple "model_version_id" being returned but I am not sure what it means for GBD or if it matters.  I do like the cleaner one liner but am nervous about ignoring errors but adding logging might help future debugging.  

I do think we need to find out what is causing this behavior but it is causing the TMREL category for exposure to not exist later when the groupby call is made.

---

We should investigate why we're getting different columns at different levels of the location hierarchy. We should ask the GBD team why this is.